.github/workflows/helm-unittest.yml

```
@@ -0,0 +1,27 @@
+  publish:
+    runs-on: ubuntu-latest
+
+    defaults:
+      run:
+        working-directory: deploy/helm
+        shell: bash
+
+    steps:
+      - name: Checkout the code
+        uses: actions/checkout@v4
+    
```

YAMLlint (1.37.1)

[error] trailing spaces

(trailing-spaces)